src/databricks/labs/remorph/reconcile/execute.py

```diff
@@ -67,7 +68,7 @@ def validate_input(input_value: str, list_of_value: set, message: str):
 def main(*argv) -> None:
     logger.debug(f"Arguments received: {argv}")
 
-    w = WorkspaceClient()
+    w = WorkspaceClient(profile="FE-EAST")
```

Comment:
There are other ways to do this.

Reply:
Removed the Invocation part here, will update in a separate PR.